modules/executors/variables.tf

```diff
@@ -94,7 +94,7 @@ variable "num_total_jobs" {
 
 variable "max_active_time" {
   type        = string
-  default     = "2h"
+  default     = "12h"
```

Comment:
this is the default we use on managed instances and dotcom - proven to work well.